crates/typst/src/foundations/int.rs

```diff
@@ -214,6 +211,50 @@ impl i64 {
             self >> shift
         }
     }
+
+    /// Converts bytes to an integer.
+    /// Bytes should be 8 bytes (64 bits) in size.
+    ///
+    /// ```example
+    /// #int.from-bytes(bytes((0, 0, 0, 0, 0, 0, 0, 1)), "little")
+    /// ```
+    #[func]
+    pub fn from_bytes(
+        /// The bytes that should be converted to an integer.
+        bytes: Bytes,
+        /// Endianness
+        /// - Little endian: the bytes are interpreted from the least significant to the most significant.
+        /// - Big endian: the bytes are interpreted from the most significant to the least significant.
```

Comment:
Seems this document does not tell users how to specify the endianness? I would suggest clearly document the acceptable strings here, instead of indicating them in examples.

Reply:
Improved the documentation in the latest commit